doc/nrf/ug_chip.rst

```diff
@@ -0,0 +1,64 @@
+.. _ug_chip:
+
+Connected Home over IP (CHIP)
+#############################
+
+.. chip_intro_start
+
+CHIP is an open-source application layer project that aims at creating a unified communication standard across smart home devices, mobile applications, and cloud services.
+It supports a wide range of existing technologies, including Wi-Fi, Thread, and Bluetooth LE, and uses IPv6-based transport protocols like TCP and UDP to ensure connectivity between different kinds of networks.
+
+.. chip_intro_end
+
+The |NCS| allows you to develop applications that are compatible with the Connected Home over IP project.
+
+Supported features
+******************
+
+The CHIP protocol implementation in |NCS| supports the following features:
+
+* Data Model layer based on ZCL (Zigbee Cluster Library).
+* CHIP commissioning procedure over Bluetooth LE, SoftAP, or Ethernet.
+  Operational certificates are used for establishing secure point-to-point communication.
+
+  .. note::
+        Commissioning over Thread is not supported.
+
+* CHIP Reliable Message Protocol (CRMP) - reliable UDP-based transport protocol.
+
+Architecture
+************
+
+CHIP defines and application layer on top of the IPv6-based transport protocols.
+This allows for routing messages regardless of the underlying physical and link layers.
+
+.. figure:: images/CHIP_IP_pyramid.png
+   :alt: CHIP architecture overview
+
+   CHIP architecture overview
+
+The CHIP application layer can be broken down into several main components, from IP framing and transport management up to Data Model structure and Application itself.
+For detailed description, see the `CHIP Protocol Overview`_ page.
+
+Integration with |NCS|
+======================
+
+The CHIP project is included in the |NCS| as one of the repositories managed with the west tool.
+However, |NCS| is also one of the platforms available in CHIP.
```

Comment:
How about adding: "That is, the code responsible for |NCS| and CHIP integration lies in the CHIP repository" or something similar?